src/Take.Blip.Builder/Actions/CreateTicket/CreateTicketAction.cs

```diff
@@ -66,12 +66,12 @@ public override async Task ExecuteAsync(IContext context, CreateTicketSettings s
                 };
             }
 
-            var createdTicket = await _helpDeskExtension.CreateTicketAsync(ticket, cancellationToken);
-            context.SetTicket(createdTicket);
+            ticket = await _helpDeskExtension.CreateTicketAsync(ticket, cancellationToken);
+            context.SetTicket(ticket);
 
             if (!string.IsNullOrWhiteSpace(settings.Variable))
             {
-                await context.SetVariableAsync(settings.Variable, createdTicket.Id, cancellationToken);
+                await context.SetVariableAsync(settings.Variable, ticket.Id, cancellationToken);
```

Comment:
I'm not sure this will help the GC. The previous value is still no collected.

Reply:
It will, now the createdTicket variable is in the global scope, that way we won't create a new variable to gc collect